README.md

```diff
@@ -0,0 +1,168 @@
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
+
+-   [Persistent cross-site scripting
+    vulnerability](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Cross-site request forgery vulnerabilities in Scriptler script
+    management](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Any user can add Scriptler scripts to build
+    configurations](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Arbitrary code execution vulnerability in rare
+    circumstances](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Any Scriptler script can be executed as part of
+    builds](https://jenkins.io/security/advisory/2017-04-10/)
```

Comment:
This is automatically generated text that was inserted to the wiki: the plugin site will fetch the latest security warnings automatically. Please remove.

Some level1 heading like "Scriptler Plugin for Jenkins" could be useful instead.